integration-tests/utils/integration-utils.js

```diff
@@ -171,8 +171,12 @@ const noSpinnersAfter = async (page, { action, debugMessage }) => {
   if (debugMessage) {
     console.log(`About to perform an action and wait for spinners. \n\tDebug message: ${debugMessage}`)
   }
-  const foundSpinner = page.waitForXPath('//*[@data-icon="loadingSpinner"]')
-  await Promise.all([foundSpinner, action()])
+  const foundSpinner = page.waitForXPath('//*[@data-icon="loadingSpinner"]', { timeout: 5000 })
+  await Promise.all([
+    // loadingSpinner is not always visible. Log but ignore error if loadingSpinner is not found
+    foundSpinner.catch(e => { console.error(e) }),
```

Comment:
Was this meant to be

```suggestion
    foundSpinner.catch(e => console.error(e)),
```

?

Reply:
Yes, Thanks.